faqs/galaxy/dataset_tool_erorrs_metadata.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Tool Errors related to Metadata
```

Comment:
hmm I thinking now that we may want to have the title as an explicit question, what do you think @Beachyesh?

Reply:
Okay, so this one has felt a little tricky. At first, I kept the title about missing Metadata, but Anne pointed out how will the user know that the error is about the missing metadata? I thought it was a fair point and hence I made the decision to keep the FAQ title slightly broader. Considering this suggestion and usual title guidelines in mind does "Incorrect or Undetected Datatype Attributes" sound like an appropriate title?

Reply:
Well, we do not know everything and this is why collaborating together is usually best!

Maybe something like: "How to find and correct tool Errors related to Metadata?"

Reply:
Yess sounds good!